packages/editor/src/client/api/index.ts

```diff
@@ -28,7 +24,7 @@ export type Scalars = {
 };
 
 export type Article = {
-   __typename?: 'Article';
+  __typename?: 'Article';
```

Comment:
Did your editor do this kind of linting? because there's a git pre commit hook that does the linting

Reply:
actually i assume it was done by IDE (JetBrains) ... i could drop this if its not correct

Reply:
yes please. All the linting and formating should be done by the pre-commit hook.